starfish/image/_filter/util.py

```diff
@@ -1,6 +1,7 @@
 from typing import Tuple, Union
```

Comment:
We need to discuss this in depth, specifically how we deal with values greater than 1.

Reply:
Here's the start of an in-depth discussion of how we and other packages deal with values greater than 1: https://docs.google.com/document/d/1TNbzOOml-sOp9PEEM7Y8LQHansxPS02flEOFyetNfv8/edit#heading=h.7kmz03ilv9in